simcon_project/users/migrations/0001_initial.py

```diff
@@ -1,10 +1,9 @@
-# Generated by Django 3.1.3 on 2020-11-24 01:51
```

Comment:
Curious why this migration file is being kept and the other's aren't?

Reply:
Let me double check on that since I thought I got rid of it

Reply:
I just checked and it's not on my branch and it's also up to date so it might be a bug on github's end?

Reply:
Oh I just checked again and it's gone 👍